resources/core/charts/service-catalog-addons/charts/brokers-ui/templates/brokers-microfrontend.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: "ui.kyma-project.io/v1alpha1"
+kind: ClusterMicroFrontend
+metadata:
+  name: brokersmicrofrontend
+  labels:
+    app:  brokersmicrofrontend
+spec:
+  displayName: Brokers
+  version: v1
+  category: Service Catalog
+  placement: namespace
+  viewBaseUrl: 'https://brokers.{{ js .Values.global.domainName }}'
+  navigationNodes:
+    - label: Brokers
+      navigationPath: brokers
+      viewUrl: ''
+      order: 3
+---
+apiVersion: "ui.kyma-project.io/v1alpha1"
+kind: ClusterMicroFrontend
+metadata:
+  name: servicebrokersmicrofrontend
+  labels:
+    app:  servicebrokersmicrofrontend
+spec:
+  displayName: Service Brokers
+  version: v1
+  category: Integration
+  placement: cluster
```

Comment:
Just a question. Does it mean cluster service brokers view uses broker-ui application now? Cause as I know those two views were separate applications before, right?

Reply:
No, it is still part of the console app itself (viewBaseUrl: 'https://console.{{ js .Values.global.domainName }}/consoleapp.html#') as it was before, but now it is optional. It can later be moved to a dedicated app or to the brokers-ui app or stay as a part of the console - we have all the options in the future.